src/simplify.jl

```diff
@@ -16,14 +16,14 @@ function simplify!(com)
     b_not_equal_to = false
```

Comment:
I'm wondering how much of this file could be useful more generally, for all CP solvers (something like https://github.com/mtanneau/MathOptPresolve.jl).

Reply:
That would definitely be useful in my opinion.

Reply:
We can think about it in greater detail later on, even MathOptPresolve is in its infancy.